pxt.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Roller Coaster Builder",
+    "version": "0.1.47",
+    "description": "",
```

Comment:
I would make sure to add a description so there's something on the card in the extensions gallery other than the picture.